docs/builtin_elements.md

```diff
@@ -831,6 +831,11 @@ against the constants below.
 * **`Escape`**
 * **`Backtab`**
 * **`Delete`**
+* **`Shift`**
+* **`Control`**
+* **`Alt`**
+* **`AltGr`**
+* **`CapsLock`**
```

Comment:
I think we also need a Key for `Command`.

Reply:
Is it not mapped to the `Home` key?

Reply:
Hm, I've never heard of that mapping. Where would that be?

AFAIK neither Winit or Qt do that.

Reply:
To clarify: With winit command is mapped to the Windows key (left and right, depending on the side), not the `Home` key.

Reply:
I added the `meta` macos command key. It works now for `winit` and `qt`. The only remarkable is that qt sends for meta / command `Qt_Key_Key_Control` and for control `Qt_Key_Key_Meta`. I can't find out what's the source of this behavior, but switching this in  `key_codes.rs` worked.

Reply:
Yes, that is an old behavior of Qt. See also https://github.com/slint-ui/slint/pull/1747#issuecomment-1291773189 . Olivier has kind of convince me that we should not follow this swap, but I think it's worth doing that in a separate patch.

Note that this swap is only done on macOS, so I think this can't be coded in `key_codes.rs` but needs to be done at run-time: Nothing to do for Qt, but with winit the existing code to produce the same behavior as Qt on macOS should be kept there perhaps (only on macOS that is).